tensorflow/tools/tf_sig_build_dockerfiles/setup.packages.sh

```diff
@@ -19,10 +19,16 @@
 # Usage: setup.packages.sh <package_list.txt>
 set -e
 
+
+if [ -n ${CUSTOM_INSTALL} ] && (source /etc/os-release && [[ ${NAME} == SLES ]]); then
```

Comment:
It might be good to support building for SLES outside of a custom install as well.
Can you add a SLES check in the non custom-install path?